encrypted-media-respec.html

```diff
@@ -1604,15 +1611,95 @@ <h2><dfn>MediaKeys</dfn> Interface</h2>
+                      <ol>
+                        <li><p>If the CDM cannot determine the <a>MediaKeyStatus</a> for <a def-id="dictionary-member">member</a>, then reject <var>promise</var> with <a def-id="NotSupportedError"></a> and abort these steps.</p></li>
+                      </ol>
+                    </li>
+                    <li><p>For each <a def-id="dictionary-member">member</a> of <var>policy</var>, run the following steps:</p>
+                      <ol>
+                        <li><p>If the CDM would block presentation of decrypted media data for <a def-id="dictionary-member">member</a>, then resolve <var>promise</var> with <a def-id="status-output-restricted"></a>.</p></li>
+                      </ol>
+                    </li>
+                    <li><p>Resolve <var>promise</var> with <a def-id="status-usable"></a>.</p></li>
+                  </ol>
+                </li>
+                <li><p>Return <var>promise</var>.</p></li>
+              </ol>
+              <table class="parameters">
+                <tbody>
+                  <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Nullable</th>
+                    <th>Optional</th>
+                    <th>Description</th>
+                  </tr>
+                  <tr>
+                    <td class="prmName">policy</td>
+                    <td class="prmType">
+                      <code>MediaKeysPolicy</code>
+                    </td>
+                    <td class="prmNullFalse">
+                      <span role="img" aria-label="True">✔</span>
+                    </td>
+                    <td class="prmOptFalse">
+                      <span role="img" aria-label="True">✔</span>
+                    </td>
+                    <td class="prmDesc">A set of policy requirements</td>
+                  </tr>
+                </tbody>
+              </table>
+              <div><em>Return type: </em><code>MediaKeyStatus</code></div>
+
+              <div>
+                <pre class="idl">
+                  dictionary MediaKeysPolicy {
+                      HDCPVersion minHdcpVersion;
+                  };
+                </pre>
+                <p>The <a>MediaKeysPolicy</a> dictionary is an object consisting of only optional properties. Each property represents a
+                  policy requirement. A policy is said to be fulfilled if the CDM would allow presentation of decrypted
+                  media data based on all of the requirements.</p>
+              </div>
+              <div>
+                <pre class="idl">
+                  enum HDCPVersion {
+                      "1.0",
+                      "1.1",
+                      "1.2",
+                      "1.3",
+                      "1.4",
+                      "2.0",
+                      "2.1",
+                      "2.2",
+                      "2.3",
```

Comment:
Just a note here. If the industry introduces newer HDCP versions, e.g. "2.4", we'll have to update the EME spec to support it. Historically we've been using Registries to deal with similar cases; I am not sure whether we want to do it this time with the additional complexity.

Reply:
@tidoust @chrisn Should I do a registry for this? It seems like [https://github.com/w3c/encrypted-media](https://github.com/w3c/encrypted-media) would have to updated in either case.

Reply:
It does add a bit of extra complexity, but we have examples in WebCodecs we can borrow from that follow the latest Registry track Process requirements, so shouldn't be too difficult. It would mean we can add new versions without updating the spec itself which could be useful once we've completed v2.

Reply:
I missed this comment a couple of weeks ago. Yes, a registry would seem appropriate, also as a way to reference what "x.y" means in practice with a link to the relevant specification.

If you follow the registry path, I note that you'll need to drop the   `enum` and rather use a `DOMString`, as [done in WebCodecs](https://www.w3.org/TR/webcodecs/#dom-audiodecoderconfig-codec).

Reply:
Will do.